### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by daniel-clayton**
* Removed migrations
* Changed url_title to slug
* FaqAdmin now inherits from SortableModelAdmin
* Added get_form method to FaqAdmin

### CODE REVIEW

File: apps/faqs/admin.py [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
-            'fields': ['page', 'question', 'slug', 'answer', 'categories', 'order']
+            'fields': ['page', 'question', 'slug', 'answer', 'categories']
         }),
         SearchMetaBaseAdmin.PUBLICATION_FIELDS,
         SearchMetaBaseAdmin.SEO_FIELDS,
     )
+    def get_form(self, request, obj=None, **kwargs):
+        form = super(FaqAdmin, self).get_form(request, obj, **kwargs)
+        try:
+            form.base_fields['page'].initial = Faq.objects.all()[0]
```
Comment (lewiscollard) [inline apps/faqs/admin.py]: I think this should be `Faqs.objects.all()`.

File: apps/faqs/admin.py [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
 
     fieldsets = (
         (None, {
-            'fields': ['page', 'question', 'url_title', 'answer', 'categories', 'order']
+            'fields': ['page', 'question', 'slug', 'answer', 'categories']
         }),
         SearchMetaBaseAdmin.PUBLICATION_FIELDS,
         SearchMetaBaseAdmin.SEO_FIELDS,
     )
+    def get_form(self, request, obj=None, **kwargs):
```
Comment (danielsamuels) [inline apps/faqs/admin.py]: Stick a space before this method.

File: apps/faqs/admin.py [hunk 3]
```diff
@@ -1,21 +1,29 @@
...
+        form = super(FaqAdmin, self).get_form(request, obj, **kwargs)
+        try:
```
Comment (danielsamuels) [inline apps/faqs/admin.py]: Space before this line.

File: apps/faqs/admin.py [hunk 4]
```diff
@@ -1,21 +1,29 @@
...
+            form.base_fields['page'].initial = Faq.objects.all()[0]
+        except IndexError:
+            pass
+        return form
```
Comment (danielsamuels) [inline apps/faqs/admin.py]: Space before this line.

**Review by danielsamuels — CHANGES_REQUESTED** [review]

File: apps/faqs/admin.py (hunk 1 above)
Comment (danielsamuels) [reply apps/faqs/admin.py]: Yup.

**Review by danielsamuels — APPROVED** [review]

File: apps/faqs/admin.py [hunk 5]
```diff
@@ -1,22 +1,33 @@
...
         }),
         SearchMetaBaseAdmin.PUBLICATION_FIELDS,
         SearchMetaBaseAdmin.SEO_FIELDS,
     )
 
+    def get_form(self, request, obj=None, **kwargs):
+        form = super(FaqAdmin, self).get_form(request, obj, **kwargs)
+
+        try:
+            form.base_fields['page'].initial = Faqs.objects.all()[0]
```
Comment (danielsamuels) [inline apps/faqs/admin.py:26]: You'll need to import Faqs to be able to use it.

**Review by danielsamuels — CHANGES_REQUESTED** [review]

**Comment by lewiscollard** [comment]
With the tiny issues now fixed, this all looks great to me.

File: apps/faqs/models.py [hunk 6]
```diff
@@ -41,7 +41,7 @@ def __unicode__(self):
         return self.title
 
     class Meta:
-        verbose_name = 'catgeory'
+        verbose_name = 'categeory'
```
Comment (danielsamuels) [inline apps/faqs/models.py]: This is still misspelt. :)

File: apps/faqs/admin.py [hunk 7]
```diff
@@ -1,22 +1,33 @@
 from cms.admin import PageBaseAdmin, SearchMetaBaseAdmin
+from suit.admin import SortableModelAdmin
```
Comment (lewiscollard) [inline apps/faqs/admin.py]: Move this below the below Django import to ensure they are sorted. :)

File: apps/faqs/models.py (hunk 6 above)
Comment (danielsamuels) [reply apps/faqs/models.py]: You don't actually need this line at all, the model is named `Category` so this value will be correct. It's just the pluralised form which would be wrong (but that's being sorted on the line below).

File: apps/faqs/admin.py [hunk 8]
```diff
@@ -1,22 +1,33 @@
...
 
-from .models import Category, Faq
+from .models import Category, Faq, Faqs
 
 
 @admin.register(Faq)
-class FaqAdmin(SearchMetaBaseAdmin):
-    prepopulated_fields = {'url_title': ('question',)}
+class FaqAdmin(SearchMetaBaseAdmin, SortableModelAdmin):
+    prepopulated_fields = {'slug': ('question',)}
```
Comment (lewiscollard) [inline apps/faqs/admin.py:10]: Put an empty line under this - class attributes should usually have an empty line underneath. (It was like this before you got to it, but we may as well do it now.)

**Comment by danielsamuels** [comment]
This app is going to be absolutely byte-perfect by the time LC and I are finished reviewing :D

**Review by danielsamuels — APPROVED** [review]

**Review by lewiscollard — APPROVED** [review]

**Comment by dan-gamble** [comment]
If you wanna get byte-perfect i'd suggest making the dangling commas consistent ;)

**Comment by danielsamuels** [comment]
Thanks for your work on this PR!

**Comment by daniel-clayton** [comment]
Np, ty!

**Pull request merged**